pkg/controller/permissions/permissions_check.go

```diff
@@ -104,6 +114,22 @@ func CheckPermissions(req router.Request, _ router.Response) error {
 			details.AppImage.Digest, appImage.Digest)
 	}
 
+	// ServiceNames of the current app level (i.e. not nested Acorns/Services)
+	scvnames := maps.Keys(details.AppSpec.Containers)
+	scvnames = append(scvnames, maps.Keys(details.AppSpec.Jobs)...)
+	scvnames = append(scvnames, maps.Keys(details.AppSpec.Services)...)
```

Comment:
Do we need to add sidecar names to this as well? I believe the "service name" (as far as permissions are concerned) for sidecars is in the format of `<parent container name>.<sidecar name>`.

Reply:
Permissions set on sidecars are "assigned" to the parent container :+1: